iphone/TitaniumKit/TitaniumKit/Sources/API/Webcolor.m

```diff
@@ -30,11 +32,45 @@ + (UIColor *)checkmarkColor
   return checkmarkColor;
 }
 
+
++ (UIColor *)semanticColorNamed:(NSString *)colorName
+{
+  if ([TiUtils isIOSVersionOrGreater:@"11.0"]) {
+    return [UIColor colorNamed:colorName];
+  }
+
+  if (semanticColors == nil) {
+    NSString *colorsPath = [NSBundle.mainBundle pathForResource:@"semantic.colors" ofType:@"json"];
+    if (![NSFileManager.defaultManager fileExistsAtPath:colorsPath]) {
```

Comment:
Do we need to worry about an encrypted version of the file?

Reply:
Are JSON resource files encrypted as well? Looking at a recent production build, it doesn't seem to.

Reply:
We don't encrypt JSON files under "Resources". So, we should be okay, but it doesn't hurt to test with a device build just in case.

(We do encrypt our internal bootstrap JSON and app properties JSON though. Our build system is capable of doing this for all JSON files if we wanted to.)